CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ if(APPLE)
     # Minimum macOS version supported by Qt 5.12
     set(CMAKE_OSX_DEPLOYMENT_TARGET 10.12 CACHE STRING "Minimum macOS version the build will be able to run on")
     if(NOT VCPKG_TARGET_TRIPLET)
-      set(VCPKG_TARGET_TRIPLET "x64-osx-min10.12")
+      set(VCPKG_TARGET_TRIPLET "x64-osx-min1012")
```

Comment:
I'm kinda iffy about this required change. Any opinion @Be-ing, @Holzhaus?

Reply:
The change was required because it was required to fix a build error after merging upstream. This commit:

https://github.com/mixxxdj/vcpkg/commit/fb78450acab419844eede708c3b30cef1a102ada

Reply:
Mhmm ok, thanks for clarifiying.